### ISSUE TITLE
SSE41.Blend unsupported when Benchmarking out of process

### ISSUE THREAD

**Issue by HurricanKai** [dotnet/BenchmarkDotNet#1421]
**.Net 5**
When using SSE41 specific Methods (Blend, PackUnsignedSaturate etc.) in Benchmarks the Benchmark cannot be run, due to a PlatformNotSupportedException.
The initial Program repors that SSE42 & SSE41 are supported (Sse41/Sse42.IsSupported are true) but the Benchmarks ran from it do still throw. (While Sse41.IsSupported is still true)
When using `DebugInProcessConfig` as config all the in process Benchmarks work fine, while the out of process do not.
I can provide a Repo to reproduce, if required (in theory reproducing this should be very simple)

**Comment by HurricanKai** [comment]
It seems I have generalized a way too quickly, sorry about that

**Comment by adamsitnik** [comment]
Hello @HurricanKai 

Could you please provide a repro?

BTW are you using the latest version (0.12.1) which added the .NET 5 support?

**Comment by adamsitnik** [comment]
@HurricanKai ping

**Comment by HurricanKai** [comment]
Sorry, I seem to have missed that last message.
When trying to reproduce this in a minimal sample it just worked.
Only in that one project have I encountered the issue. The versions and such are the same, as far as I can tell. I've just copied the files to a new directory, and there it seems to work. Maybe I somehow screwed up some Intrinsics availability in that specific Artifacts folder...
Doesn't seem to be an Issue with BenchmarkDotNet, it does what it's supposed to.
